examples/01-basic/01-minimal/package.json

```diff
@@ -23,15 +22,6 @@
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.9",
     "@vitejs/plugin-react": "^4.3.1",
-    "eslint": "^8.10.0",
     "vite": "^5.3.4"
-  },
-  "eslintConfig": {
```

Comment:
intentional to remove eslint from examples, or is it solved in a different way?

Reply:
eslint is global, so still works locally

the referenced config file never worked for the sandboxes, so better to just remove altogether